tests/init/neg/enum.check

```diff
@@ -0,0 +1,9 @@
+-- Error: tests/init/neg/enum.scala:4:8 --------------------------------------------------------------------------------
+4 |        NoExplanationID // error
+  |        ^
+  |        Promoting the value to fully-initialized is unsafe. May only use initialized value as method arguments
+  |
+  |        The unsafe promotion may cause the following problem(s):
+  |
+  |        1. Calling the external method method name may cause initialization errors. Calling trace:
+  |         -> NoExplanationID // error	[ enum.scala:4 ]
```

Comment:
Maybe we can add another example for the warning. You can compile the file as follows in sbt:

```
sbt > scalac -d out -Xprint:initChecker -Ysafe-init tests/init/neg/enum.scala
```

With desugared code, it's easier to see what's happening.

Reply:
The desugared code from Xprint contains many annotations and cannot pass the compilation. What modifications do I need?

Reply:
The from -Xprint is meant to help you create examples with the same warning but without using enums.